build.sh

```diff
@@ -52,11 +52,17 @@ SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pw
 
 # Copy Thunder project to Dockerfile context if project path is provided
 if [ "${PROJECT_PATH}" != "" ]; then
+  ADDITIONAL_DEPS="drush/drush thunder/thunder_performance_measurement thunder/testsite_builder"
+  # @todo move from thunder_demo to thunder_testing_demo
+  # if [ "${PROFILE}" == "thunder" ]; then
+  #   ADDITIONAL_DEPS="${ADDITIONAL_DEPS} thunder/thunder_testing_demo"
+  # fi
+
   rm -rf "${SCRIPT_DIRECTORY}/www"
   cp -R "${PROJECT_PATH}" "${SCRIPT_DIRECTORY}/www"
   # Compose project to ensure dependencies are correct.
   cd "${SCRIPT_DIRECTORY}/www"
-  composer require drush/drush thunder/thunder_performance_measurement thunder/testsite_builder
+  composer require "${ADDITIONAL_DEPS}"
```

Comment:
Why not just using 2 require commands. Leave this one and an additional for profile=thunder

Reply:
Well to eventually cope with 
```
#else
  # @todo Ensure a the minimum of build tools.
  # cd "${SCRIPT_DIRECTORY}/www"
  # composer require drush/drush drupal/console
  # cd "${SCRIPT_DIRECTORY}"
```